docs/4.0/i18n/zh-Hans/getting-started/kuberentes-life-cycle.md

```diff
@@ -26,7 +26,7 @@ $ sealos run labring/kubernetes:v1.25.0 labring/calico:v3.24.1 \
 | --masters |  192.168.0.2 | kubernetes master 节点地址列表 |
 | --nodes | 192.168.0.3 | kubernetes node 节点地址列表 |
 | --ssh-passwd | [your-ssh-passwd] | ssh 登录密码 |
-|image | labring/kubernetes:v1.25.0 | kubernetes 镜像 |
+|kubernetes | labring/kubernetes:v1.25.0 | kubernetes 镜像 |
```

Comment:
This is `cluster image` not kubernetes, it also can be `mysql:8.0` or other applications image